Automattic/jetpack
# Paywall: Use Subscriber Login block for the login link

Fixes https://github.com/Automattic/jetpack/issues/36289

## Proposed changes:

It renders the Subscriber Login block instead of creating the login link from scratch.

More cleanup is already possible thanks to this change but I'll do it when working on https://github.com/Automattic/jetpack/issues/36307.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Be not logged in to wp.com
* Open blog post with the paywall block
* Click "Already a subscriber?" link
* Make sure you were redirected to the "Email me a login link" page
* Log in using the login link
* Make sure you were logged in correctly
